Dockerfile

```diff
@@ -0,0 +1,39 @@
+#
+# FM-Frontend-API Docker File
+# Builds a docker image to run the FM frontend services
+#
+
+FROM node:slim
+
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates
```

Comment:
Node.js is built with GYP and thus some packages require it when they are installed  https://github.com/joyent/node/wiki/installation#building-prerequisites. I'm not sure that any of the specific packages we are using actually require it though, I'll check.